app/history_v2_player_support.go

```diff
@@ -0,0 +1,56 @@
+package app
+
+import (
+	"net/http"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/topfreegames/mqtt-history/logger"
+	"github.com/topfreegames/mqtt-history/mongoclient"
+
+	"github.com/labstack/echo"
+	"github.com/topfreegames/mqtt-history/models"
+)
+
+func HistoriesV2PSHandler(app *App) func(c echo.Context) error {
+	return func(c echo.Context) error {
+		c.Set("route", "HistoriesV2PlayerSupport")
+		topic := c.ParamValues()[0]
+
+		userID, playerId, _, limit, isBlocked := ParseHistoryQueryParams(c, app.Defaults.LimitOfMessages)
+		authenticated, _, err := IsAuthorized(c.StdContext(), app, userID, topic)
```

Comment:
Thinking that this is an "admin" route, we could simplify this Authorization procedure, what do you think?

Reply:
Yes, it makes sense Dani! Thanks